src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -87,12 +76,6 @@ public final class CreateVariantIngestFiles extends VariantWalker {
             optional = true)
     public boolean enableVet = true;
 
-    @Argument(fullName = "enable-pet",
-            shortName = "ep",
-            doc = "write pet data",
-            optional = true)
-    public boolean enablePet = true;
```

Comment:
since we from the default as true here, maybe we make it default to true on ref ranges above?